### DISCUSSION TITLE
Hey everyone!

### DISCUSSION THREAD

**Discussion by TarangGarlapally** [OpenMined/courses#35] [Introduce Yourself]
Hey everyone!
I am Tarang, an undergrad student from India. Looking forward to learning and working with the skills.

**Comment by yemikifouly** [comment]
Hello Taran! 🙂 Welcome to the discussion board! Have you started the course yet?

**Reply by TarangGarlapally** [reply]
Thanks! Yeah, I've started the course.

**Reply by yemikifouly** [reply]
I hope you enjoy it very much and don't forget to post in the board any question that comes up. 🙂

**Reply by TarangGarlapally** [reply]
Yeah sure!
